src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java

```diff
@@ -80,6 +80,7 @@ public class NotificationWebSocketHandler implements WebSocketHandler {
     static final String USER_TAG = "user";
 
     private final ObjectMapper jacksonObjectMapper;
+    public static final String HEADER_PARAMS_NAME = "paramsName";
```

Comment:
why public ?
move it just near the other HEADER_*** constants definition maybe ?

Reply:
Fixed.